src/components/v2/Tooltip/Tooltip.tsx

```diff
@@ -3,9 +3,9 @@ import { Global } from '@emotion/react';
 import MuiTooltip, { TooltipProps } from '@mui/material/Tooltip';
 import { useStyles } from './styles';
 
-type ITooltipProps = TooltipProps;
+export { type TooltipProps } from '@mui/material/Tooltip';
 
-export const Tooltip = ({ children, ...rest }: ITooltipProps) => {
+export const Tooltip = ({ children, ...rest }: TooltipProps) => {
```

Comment:
I think we should consistently type function components with `React.FC`

Reply:
Yep definitely, didn't realize that.